packages/strapi-design-system/src/DateTimePicker/__tests__/DateTimePicker.e2e.js

```diff
@@ -0,0 +1,137 @@
+const { injectAxe } = require('axe-playwright');
+
+const { test, expect } = require('@playwright/test');
+
+test.describe.parallel('DateTimePicker', () => {
+  test.describe('without initial data', () => {
+    test.beforeEach(async ({ page }) => {
+      // This is the URL of the Storybook Iframe
+      await page.goto('/iframe.html?id=design-system-components-datetimepicker--base&viewMode=story');
+      await injectAxe(page);
+    });
+    test('change the date and see if also the time will be changed when both are empty', async ({
+      page,
+      browserName,
+    }) => {
+      test.skip(browserName === 'webkit', 'Still working on it');
+      await page.focus('[aria-labelledby="datetime-label"] > div > div:nth-child(2) > div > div');
+
+      await page.keyboard.press('Tab');
+      await page.keyboard.press('Enter');
+      await page.keyboard.press('ArrowDown');
+      await page.keyboard.press('Tab');
+      await page.keyboard.press('Tab'); // move to the Su column
+      await page.keyboard.press('ArrowDown');
+      await page.keyboard.press('ArrowDown');
+      await page.keyboard.press('ArrowDown'); // move to the third Sunday of the month
+      await page.keyboard.press('Enter'); // to select the date
```

Comment:
I'm not sure to understand why you need to do all these keyboard press? Can't you just do that with a click ?

Reply:
it was to check the navigation with keyboard, but ok, I can only try with the click

Reply:
Basically, I have implemented the navigation with the keyboard because I have found also some bugs inside the Component code with it, but if you are not ok with it I can refactor the code using a simple click event

Reply:
done